bentoctl/deployment_config.py

```diff
@@ -90,8 +91,9 @@ def get_bento_metadata(bento_path: str) -> dict:
 
 
 class DeploymentConfig:
+    bento: Bento
+
     def __init__(self, deployment_config: t.Dict[str, t.Any]):
-        self.bento = None
```

Comment:
Why did we move this up? `usage_stats.py` expects bento is always there.

Reply:
Since we are setting bento via `setup_bento` (which we will refactor latter), no need for bento to be init to None. 

By init bento to None, type hinting and LSP just go to 🗑️

Reply:
We ran into an error during `bentoctl init` if bento is not initialized to `None`.

```
AttributeError: 'DeploymentConfig' object has no attribute 'bento'
```

https://github.com/bentoml/bentoctl/blob/main/bentoctl/utils/usage_stats.py#L37

Reply:
i see